.buildkite/publishCategories.ts

```diff
@@ -0,0 +1,102 @@
+import { parse } from "npm:yaml";
```

Comment:
Should we import from `deno.land/x`?

```suggestion
import { parse } from "https://deno.land/x/yaml@2.3.4/mod.ts"
```

We might benefit from from deno's security model and specific versioning.